lib/gtfsplus/selectors/index.js

```diff
@@ -0,0 +1,42 @@
+import { createSelector } from 'reselect'
```

Comment:
Maybe we can also try to target selectors for the next round of flow-typing.

Reply:
Yea, that should be a relatively small effort.